site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Kubernetes Troubleshooting"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page helps developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster, which is running on AWS using EKS.
+
+## Set up a local environment to access Kubernetes (K8s)
+
+1. Make sure you have `kubectl` version 1.10 or later installed. If not, follow [these instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+2. Use your OneLogin account to retrieve AWS Keys for the main Mattermost AWS account following [these instructions](../../onelogin-aws)
+
+NOTE: When using Onelogin-aws account for Kubernetes configuration, please select the main Mattermost AWS account.
+
+### Configure kubectl for Amazon EKS
+
+Follow [these instructions](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html)
+
+### Create a kubeconfig for Amazon EKS
+
+Follow [these instructions](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)
+
+NOTE: Please talk with Carlos to get the cluster name.
+
+
+### Check if you can see the pods
+
+To check if you can see the pods in the K8s cluster, you can check doing this:
+
+```Bash
+$ kubectl get po -n community
```

Comment:
When I get here I get `error: the server doesn't have a resource type "po"`
Similar ally `kubectl get svc` gives `error: the server doesn't have a resource type "svc"`
I think it's an auth problem because I get:
```
> kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T21:04:45Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
error: You must be logged in to the server (the server has asked for the client to provide credentials)
```

I have aws CLI authenticated properly.

Reply:
i will check this later this week

Reply:
done